src/Pos/Web/Doc.hs

```diff
@@ -139,3 +145,9 @@ instance ToSample GodTossingStage where
                       , ("Shares stage", SharesStage)
                       , ("Ordinary stage", OrdinaryStage)
                       ]
+
+instance ToSample TxOut where
+    toSamples _ =
+        [ ("No coins",  TxOut (makePubKeyAddress (mkKey "a")) (mkCoin 0))
```

Comment:
It's probably a bad example in this case, because `Utxo` can't contain such `TxOut`s. It's a good example in isolation, but not for `utxo` endpoint. Anyway, it's minor.